### PR TITLE
Update devtools version and resource priority handling

### PR DESCRIPTION
**Pull request by deepanjanroy**
Working on #454. `lib/traces/devtools-timeline-model` unit tests are currently failing and need to be updated to match the new devtools code. Can someone confirm whether I can just change the tests, or whether we need to adapt our code elsewhere?

### CODE REVIEW

**Comment by brendankenny** [comment]
@paulirish brought those tests over from https://github.com/paulirish/devtools-timeline-model, which I think will also need to have its tests updated if we want everything on the same chrome-devtools-frontend version

**Comment by paulirish** [comment]
I will update these tests.

File: lighthouse-core/package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
   "homepage": "https://github.com/googlechrome/lighthouse#readme",
   "dependencies": {
     "axe-core": "^1.1.1",
-    "chrome-devtools-frontend": "1.0.381789",
+    "chrome-devtools-frontend": "1.0.400960",
```
Comment (brendankenny) [inline lighthouse-core/package.json]: can you bump this to 1.0.401423 to match what's going into g3?

**Comment by paulirish** [comment]
Merging and immediately addressing the tests.

Thanks for this, deep! kickass.

**Pull request merged**